docs/user-guide/rs.rst

```diff
@@ -0,0 +1,230 @@
+.. include:: /content/nemo.rsts
+
+.. _model-aligner-rs:
+
+Model Alignment by Rejection Sampling
+@@@@@@@@@@@@@@@@@@@@@@@
+
+In this tutorial, we will go through the process of aligning a NeMo framework model using rejection sampling. These can be models such as LLaMa2 or Mistral. Our scripts will work the same way.
```

Comment:
fix capitalization for NeMo Framework. Also suggested rewrite for clarity.

In this tutorial, we will guide you through the process of aligning a NeMo Framework model using rejection sampling. This method can be applied to various models, including LLaMa2 and Mistral, with our scripts functioning consistently across different models.

Reply:
Added the rewrite.